app/models/case_contact.rb

```diff
@@ -21,10 +19,40 @@ class CaseContact < ApplicationRecord
   ].freeze
 
   CONTACT_MEDIUMS = %w[in-person text/email video voice-only letter].freeze
-  enum contact_type: CONTACT_TYPES.zip(CONTACT_TYPES).to_h
+  # enum contact_type: CONTACT_TYPES.zip(CONTACT_TYPES).to_h
+
+  validates :contact_types, presence: true
+  validate :contact_types_included
+
+  def contact_types_included
+    contact_types&.each do |contact_type|
+      errors.add(:contact_types, :invalid) unless CONTACT_TYPES.include? contact_type
+    end
+  end
+
+
+  # @return [Array<String>]
+  def humanized_contact_types
+    contact_types&.map { |ct| ct.humanize.titleize }
```

Comment:
I think that because of how rails associations work, this should never be null, but it could be an empty array.

Reply:
WE THOUGHT SO TOO but it turned out to be a necessary `&` :woman_shrugging: